projects/iio_adpd1080/Makefile

```diff
@@ -0,0 +1,10 @@
+# Uncomment to use the desired platform
+# PLATFORM = xilinx
+# PLATFORM = altera
+PLATFORM = aducm3029
```

Comment:
Only works for aducm3029? 
If so, in parameters.h you should have only aducm3029 stuff